swan-lake/case-studies/wso2-vs-ballerina.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Move the case-studies directory to the root level

Reply:
Fixed.